lib/promise.js

```diff
@@ -34,7 +24,6 @@ var SequelizePromise = function (resolver) {
 
   promise.seqResolve = this.seqResolve;
   promise.seqReject = this.seqReject;
```

Comment:
@janmeier do you recall why we are saving these properties? Do we need them after removing emit support?